spec/controllers/hyrax/templates_controller_spec.rb

```diff
@@ -30,7 +30,8 @@
       it 'redirects to index' do
         delete :destroy, params: { id: template.name }
 
-        expect(response).to render_template :index
+        expect(response)
+          .to redirect_to "/templates?locale=en&notice=Deleted+#{template.name}"
```

Comment:
I'm not sure. I think I had a similar issue when trying to update old MIRA tests, but just ended up doing something similar.